### PR TITLE
Responsive interface and styling

### PR DESCRIPTION
**Pull request by nyashabryan**

### CODE REVIEW

File: cspatients/templates/cspatients/patient.html [hunk 1]
```diff
@@ -82,8 +85,21 @@
...
+            }else{
+                $("#patient-details-table").removeClass("table-sm");
+            };
+        };
+
+        setTableClass();
+    });
+</script>
 
 {% endblock %}
```
Comment (sewagodimo) [inline cspatients/templates/cspatients/patient.html]: add an empty line at the end of a file

File: README.md [hunk 2]
```diff
@@ -1,2 +1,12 @@
...
+
+## Running
+
+1. Set up redis on docker port 
+
+sudo dockerrun -p 6379:6379 -d redis:2.8
+
+2. Run the server.
+
+python3 manage.py runserver
```
Comment (sewagodimo) [inline README.md:12]: look at the read me on this repository (https://github.com/praekelt/molo-gem)  for a guide on how we add readmes. You can make those changes in a separate PR

**Review by sewagodimo — COMMENTED** [review]
just some comment, it looks good

**Review by KaitCrawford — APPROVED** [review]
:+1:
